radix-engine/tests/abi/src/lib.rs

```diff
@@ -25,12 +25,12 @@ blueprint! {
 }
 
 #[no_mangle]
-pub extern "C" fn AbiComponent2_main(_input: *mut u8, _input2: *mut u8) -> *mut u8 {
+pub extern "C" fn AbiComponent2_main(_input: *mut u8) -> *mut u8 {
```

Comment:
Still `Blueprint_main`? Are we supporting a hybrid mode?

Reply:
ah, it's just another level of indirection, ie every `fn_ident` must map to some extern function (in this case it is to the same function).